numpy/core/tests/test_multiarray.py

```diff
@@ -2251,7 +2251,7 @@ def test__deepcopy__(self, dtype):
     def test__deepcopy__catches_failure(self):
         class MyObj:
              def __deepcopy__(self, *args, **kwargs):
-                 return Exception
+                raise Exception
 
         arr = np.array([1, MyObj(), 3], dtype='O')
         arr.__deepcopy__({})
```

Comment:
This should float up the exception, so you could not use `Exception` above maybe (maybe AssertionError), not that it matters much.  What matters is wrapping this into:
```
with pytest.raises(Exception):  # (or a more exact Exception)
    arr.__deepcopy__({})
```

Reply:
Got it! I think I was just overthinking things a bit. Just made the changes now; appreciate the help :raised_hands: